### PR TITLE
Fix association Event - workshop, event - sponsor. Create test for sp…

### PR DESCRIPTION
**Pull request by PauladO**
…onsors-event association

### CODE REVIEW

**Comment by Cesurhan** [comment]
Confirmed.

File: app/models/workshop.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class Workshop < ApplicationRecord
+  belongs_to :event
```
Comment (arnoFleming) [inline app/models/workshop.rb]: The workshop is the topic that we teach during an event. so a workshop has many events. and an event belongs to a workshop

File: app/models/event.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 class Event < ApplicationRecord
-  belongs_to :workshop
+  has_one :workshop
```
Comment (arnoFleming) [inline app/models/event.rb]: The workshop is the topic that we teach during an event. so a workshop has many events. and an event belongs to a workshop

Comment (arnoFleming) [inline app/models/event.rb]: Please add tests for all relationships

Comment (arnoFleming) [inline app/models/event.rb]: of all models

File: spec/models/event_spec.rb [hunk 3]
```diff
@@ -1,5 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "association with sponsor" do
+    # let(:event) { create :event, sponsor: :sponsor }
+    let!(:sponsor) { create :sponsor }
```
Comment (arnoFleming) [inline spec/models/event_spec.rb]: a `let` would be enough atm

**Comment by arnoFleming** [comment]
please rebase

File: spec/models/workshop_spec.rb [hunk 4]
```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe Workshop, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "association" do
+    let (:sponsor) { create :sponsor }
```
Comment (arnoFleming) [inline spec/models/workshop_spec.rb]: dont put a space between `let` and the parenthesis. Do it like on the next line!

**Comment by PauladO** [comment]
awesome sauce

**Pull request merged**